lib/Transaction/Request/TransactionCapture.php

```diff
@@ -46,6 +55,12 @@ public function getPayload()
             $payload['metadata'] = $this->metadata;
         }
 
+        if ($this->splitRules instanceof SplitRuleCollection) {
```

Comment:
Se você usar type hint na assinatura do construtor pode evitar esse if. Se usar o type hint e algo diferente de `SplitRuleCollection` for passado por parâmetro um erro vai ser disparado.

Reply:
Já existe esse tratamento ao chamar o método, aqui: https://github.com/pagarme/pagarme-php/blob/984703cef776e5eff0f01e4d5762f3620e535c83/lib/Transaction/TransactionHandler.php#L146